packages/fxa-content-server/server/lib/amplitude.js

```diff
@@ -82,6 +82,7 @@ const VIEW_ENGAGE_SUBMIT_EVENT_GROUPS = {
   signup: GROUPS.registration,
   sms: GROUPS.connectDevice,
   subscribe: GROUPS.sub,
+  support: GROUPS.subSupport,
```

Comment:
Can you add test cases for the new view, engage and submit events to `tests/server/amplitude.js`? Ideally that module should have a test case for every event that we emit, although I think that practice may have slipped a bit recently.

Reply:
Yup, added.